src/Domain/PayPalConfig.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class PayPalConfig {
+
+	private $payPalAccountAddress;
+	private $payPalBaseUrl;
+	private $notifyUrl;
+	private $returnUrl;
+	private $cancelUrl;
+	private $itemName;
+
+	private function __construct( string $payPalAccountAddress, string $payPalBaseUrl, string $notifyUrl,
+								 string $returnUrl, string $cancelUrl, string $itemName ) {
+		$this->payPalAccountAddress = $payPalAccountAddress;
+		$this->payPalBaseUrl = $payPalBaseUrl;
+		$this->notifyUrl = $notifyUrl;
+		$this->returnUrl = $returnUrl;
+		$this->cancelUrl = $cancelUrl;
+		$this->itemName = $itemName;
+	}
+
+	/**
+	 * @param String[] $config
+	 * @return PayPalConfig
+	 */
+	public static function newFromConfig( array $config ) {
```

Comment:
If we add constants for these keys, then we avoid a class of errors and make navigation and refactoring easier.